apps/next/src/pages/Approve/components/ActionDetails/btc/components/BtcNetworkFeeWidget/types.ts

```diff
@@ -0,0 +1 @@
+export type BtcFeePreset = 'slow' | 'normal' | 'fast';
```

Comment:
Question. Why are we introducing a mapping on the type level for the values from the bitcoin vm-module? Should this be simply a mapping on the button text level? Feels like it would require less mental strain to follow what slow normal and fast means

Reply:
Yup, renamed those to match the keys in `NetworkFees` object.